packages/sdk/src/browser-wallets/xverse/__tests__/index.spec.ts

```diff
@@ -0,0 +1,150 @@
+// @vitest-environment happy-dom
+import type { GetAddressOptions, GetAddressResponse } from "sats-connect";
+import * as satsConnect from "sats-connect";
+
+import { WalletAddress } from "../../types";
+import { getAddresses, isInstalled } from "..";
+
+vi.mock("sats-connect", async (originalImport) => {
+  const mod = (await originalImport()) as typeof satsConnect;
+  return {
+    __esModule: true,
+    ...mod,
+  };
+});
+
+describe("Xverse Wallet", () => {
+  afterEach(() => {
+    vi.unstubAllGlobals();
+  });
+
+  describe("isInstalled", () => {
+    test("should return true if installed", () => {
+      vi.stubGlobal("BitcoinProvider", {});
+      expect(typeof window).not.toBeUndefined();
+      expect(isInstalled()).toBeTruthy();
+    });
+
+    test("should return false if not installed", () => {
+      expect(typeof window).not.toBeUndefined();
+      expect(isInstalled()).toBeFalsy();
+    });
+  });
+
+  describe("getAddresses", () => {
+    afterEach(() => {
+      vi.unstubAllGlobals();
```

Comment:
remove this as unstub is being performed already

Reply:
Removed